netweaver/setup/shared_disk.sls

```diff
@@ -0,0 +1,52 @@
+{%- from "netweaver/map.jinja" import netweaver with context -%}
+{% set host = grains['host'] %}
+
+{% for node in netweaver.nodes if host == node.host and node.shared_disk_dev is defined %}
+
+{% if node.init_shared_disk is defined and node.init_shared_disk == True %}
+
+label_partition:
+  cmd.run:
+    - name: /usr/sbin/parted -s {{ node.shared_disk_dev }} mklabel gpt
```

Comment:
@arbulu89 AFAIK there is a salt state for that, so we could make it idempotent. I am ok to change it in future PR though.

Reply:
Ok, I will have look to that